src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Metadata/ActivitiesListMetadataVisitor.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ActivityBundle\Infrastructure\Sulu\Metadata;
+
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\ListMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\ListMetadataVisitorInterface;
+
+/**
+ * @internal this class is internal create an own Visitor when you want to manipulated the metadata
+ */
+class ActivitiesListMetadataVisitor implements ListMetadataVisitorInterface
```

Comment:
would be nice to adjust the comment in the `activities` list configuration to use `ActivitiesListMetadataVisitor` instead of `ActivityListMetadataProvider`

Reply:
updated!